libs/banzaicloud-bank-vaults/config.jsonnet

```diff
@@ -9,7 +11,7 @@ config.new(
     {
       output: v.version,
       prefix: '^com\\.banzaicloud\\.vault\\..*',
-      crds: ['https://github.com/banzaicloud/bank-vaults/raw/'+v.tag+'/charts/vault-operator/crds/crd.yaml'],
+      crds: ['https://raw.githubusercontent.com/'+v.tag+'/charts/vault-operator/crds/crd.yaml'],
```

Comment:
This link was no longer viable, had to change to `raw.githubusercontent.com` instead.